config/db.js

```diff
@@ -7,7 +7,7 @@ const connectDB = async () => {
       // useNewUrlParser: true,
       // useUnifiedTopology: true,
     });
-    console.log('Connected to MongoDB');
+    console.log(`Connected to MongoDB at ${config.MONGO_URI}`);
```

Comment:
I've changed the naming in config, so it should be config.mongoUri instead of MONGO_URI.